tower-layer/src/identity.rs

```diff
@@ -3,21 +3,25 @@ use std::fmt;
 
 /// A no-op middleware.
 ///
-/// When wrapping a `Service`, the `Identity` layer returns the provided
+/// When wrapping a [`Service`], the [`Identity`] layer returns the provided
```

Comment:
nit/TIOLI: does this type's docs _need_ to link to its own definition? you're already reading its docs...

Reply:
Yeah I did consider that as well however I don’t think it hurts and it’s just a single pair of `[]`. Also made it easier to search for everything inside backticks and make them into links.